web/src/components/molecules/Common/Header/index.tsx

```diff
@@ -56,76 +56,68 @@ const HeaderMolecule: React.FC<Props> = ({
-          type: "divider",
-        },
-        {
-          label: t("Workspaces"),
-          key: "workspaces",
-          type: "group",
-          children: workspaces
-            ?.filter(workspace => workspace.id !== personalWorkspace?.id)
-            ?.map(workspace => ({
-              label: (
-                <Tooltip title={workspace.name} placement="right">
-                  <MenuText>{workspace.name}</MenuText>
-                </Tooltip>
-              ),
-              key: workspace.id,
-              icon: <UserAvatar username={workspace.name} size="small" shape="square" />,
-              style: { paddingLeft: 0, paddingRight: 0 },
-              onClick: () => handleWorkspaceNavigation(workspace.id),
-            })),
-        },
-        {
-          label: t("Create Workspace"),
-          key: "new-workspace",
-          icon: <Icon icon="userGroupAdd" />,
-          onClick: onWorkspaceModalOpen,
-        },
-      ]}
-    />
-  );
-
-  const AccountMenu = (
-    <HeaderMenu
-      items={[
-        {
-          label: t("Account Settings"),
-          key: "account-settings",
-          icon: <Icon icon="user" />,
-          onClick: onNavigateToSettings,
-        },
-        {
-          label: t("Logout"),
-          key: "logout",
-          icon: <Icon icon="logout" />,
-          onClick: logout,
-        },
-      ]}
-    />
-  );
+  const WorkspacesItems: MenuProps['items']  = [
+    {
+      label: t("Personal Account"),
+      key: "personal-account",
+      type: "group",
+      children: workspaces
+        ?.filter(workspace => workspace.id === personalWorkspace?.id)
+        ?.map(workspace => ({
+          label: (
+            <Tooltip title={workspace.name} placement="right">
+              <MenuText>{workspace.name}</MenuText>
+            </Tooltip>
+          ),
+          key: workspace.id,
+          icon: <UserAvatar username={workspace.name} size="small" />,
+          style: { paddingLeft: 0, paddingRight: 0 },
+          onClick: () => handleWorkspaceNavigation(workspace.id),
+        })),
+    },
+    {
+      type: "divider",
+    },
+    {
+      label: t("Workspaces"),
+      key: "workspaces",
+      type: "group",
+      children: workspaces
+        ?.filter(workspace => workspace.id !== personalWorkspace?.id)
+        ?.map(workspace => ({
+          label: (
+            <Tooltip title={workspace.name} placement="right">
+              <MenuText>{workspace.name}</MenuText>
+            </Tooltip>
+          ),
+          key: workspace.id,
+          icon: <UserAvatar username={workspace.name} size="small" shape="square" />,
+          style: { paddingLeft: 0, paddingRight: 0 },
+          onClick: () => handleWorkspaceNavigation(workspace.id),
+        })),
+    },
+    {
+      label: t("Create Workspace"),
+      key: "new-workspace",
+      icon: <Icon icon="userGroupAdd" />,
+      onClick: onWorkspaceModalOpen,
+    },
+  ];
+
+  const AccountItems: MenuProps['items'] = [
```

Comment:
Can you please wrap this array with a useMemo hook.

```suggestion
  const AccountItems: MenuProps['items'] = useMemo(() => [
```